DEVELOPERS.md

```diff
@@ -1,11 +1,10 @@
 # Clover
 
-Clover is a control plane and web console program for managing virtual
-machines and other programs.
+Clover is the codename for Ubicloud's software, which consists of the control plane, data plane, and web console program for managing virtual machines and other applications.
```

Comment:
I think `clover` is the only controlplane and web console. `rhizome` is the dataplane. cc @fdr

Reply:
Hmm, good point. Maybe it's just me, or maybe we can clarify that a little bit more then.

Reply:
yeah I guess that's a bit murky, because clover and rhizome (which is a type of root) are tightly coupled, and in the same repo, and clover uploads rhizome, but is it really a project apart? not so much.

I suppose I consider rhizome part of clover, a component.

Reply:
but yes, it is a code name